tags/tags.toml

```
@@ -813,3 +813,16 @@ The filter for collectors has moved into the options:
 + const collector = message.createReactionCollector({ filter, ...options });
 ```
 """
+
+[event-deprecation]
+keywords = ["event-deprecation", "event deprecation", "messageCreate", "interactionCreate"]
+content = """
+On master branch, `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
+Don't worry - the old names still work, but you'll receive a Deprecation Warning until you switch over.
```

```suggestion
The `message` and `interaction` events were renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
The old events still work, but you'll receive a deprecation warning until you switch over.

```

Not sure why you didn't take the suggested changes with you when re-creating.